lib/cocina/models/mapping/to_mods/title.rb

```diff
@@ -11,30 +11,29 @@ class Title # rubocop:disable Metrics/ClassLength
           TAG_NAME = Cocina::Models::Mapping::FromMods::Title::TYPES.invert.merge('activity dates' => 'date').freeze
           NAME_TYPES = ['name', 'forename', 'surname', 'life dates', 'term of address'].freeze
 
+          def self.write(...)
+            new(...).write
+          end
```

Comment:
I simplified the class method definition so we can manage params and documentation thereof in one spot (`#initialize`).